rllib/algorithms/dqn/torch/default_dqn_torch_rl_module.py

```diff
@@ -216,10 +270,13 @@ def _qf_forward_helper(
                 # Center the advantage stream distribution.
                 centered_af_logits = af_dist_output["logits"] - af_dist_output[
                     "logits"
-                ].mean(dim=1, keepdim=True)
+                ].mean(dim=-1, keepdim=True)
                 # Calculate the Q-value distribution by adding advantage and
                 # value stream.
-                qf_logits = centered_af_logits + vf_outs.unsqueeze(dim=-1)
+                qf_logits = centered_af_logits + vf_outs.view(
+                    -1, *((1,) * (centered_af_logits.dim() - 1))
+                )
+                # qf_logits = centered_af_logits + vf_outs.unsqueeze(dim=-1)
```

Comment:
remove?

Reply:
Same as above. I will check, if all combis run. If so, I will remvoe this.